doc/release-notes/0.37/0.37.md

```diff
@@ -117,20 +117,52 @@ on x64 platforms, but builds with OpenJDK 17 and later also require more stack s
-<td valign="top">Using the <tt>-XX:+ShowHiddenFrames</tt> option in an OpenJ9 release that is built with OpenJDK 18 and later causes errors due to underlying problems in OpenJDK</td>
+<td valign="top">Using the <tt>-XX:+ShowHiddenFrames</tt> option in an OpenJ9 release that is built with OpenJDK 18 and later causes errors due to underlying problems in OpenJDK.</td>
 <td valign="top">All platforms</td>
 <td valign="top">Wrong exception might be thrown when using the Reflection API.</td>
 <td valign="top">Avoid using the <tt>-XX:+ShowHiddenFrames</tt> option with OpenJDK 18 and later.</td>
 </tr>
 
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/16728">#16728</a></td>
+<td valign="top">Virtual threads may cause slowness and errors under heavy workloads due to underlying problems in OpenJ9.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Native and heap OOM errors along with performance regression might be seen when an application is scaled to 1 million+ virtual threads.</td>
+<td valign="top">Use <tt>-Xnocompressedrefs</tt> to mitigate the native OOM error. Increase <tt>-Xmx</tt> to mitigate the heap OOM error. Both options will also help mitigate the slowness.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/17163">#17163</a>, <a href=https://github.com/eclipse-openj9/openj9/issues/17181>#17181</a></td>
+<td valign="top">Virtual threads may trigger intermittent errors under heavy workloads due to underlying problems in OpenJ9.</td>
+<td valign="top">All platforms</td>
+<td valign="top">The intermittent errors might manifest as an assertion or a hang under heavy workload of virtual threads. These errors may never be seen due to the very low failure rate.</td>
+<td valign="top">Use a different GC policy to mitigate these errors.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/17119">#17119</a>, <a href=https://github.com/eclipse-openj9/openj9/issues/17120>#17120</a></td>
+<td valign="top">Virtual threads may trigger an intermittent exception due to underlying problems in OpenJ9.</td>
+<td valign="top">All platforms</td>
+<td valign="top">IllegalMonitorStateException might be thrown intermittently while trying to release a ReentrantLock. This exception may never be seen due to the very low failure rate.</td>
+<td valign="top">Disable JIT inlining to mitigate this exception.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/16756">#16756</a></td>
+<td valign="top">Virtual threads may trigger an intermittent failure due to underlying problems in OpenJ9.</td>
+<td valign="top">All platforms</td>
+<td valign="top">An intermittent segmentation fault might occur due to corruption in the Java stack of a virtual thread's carrier thread. This failure may never be seen due to the very low failure rate.</td>
```

Comment:
"intermittent" implies the VM survives the first fault to encounter a second; perhaps the issue should be described as a "rare" situation.